v1-2/learn/how-to-generate-code-for-protocol-buffers.md

```diff
@@ -53,18 +53,21 @@ message HelloResponse {
 }
 ```
 
-* You can run the following command to generate the client/service stub and service template.
+* You can run the following command to generate the service template file.
 ```
 $ ballerina grpc --input helloworld_service.proto --mode service --output service
 ```
-Once you execute the command, the stub file(`helloworld_service_pb.bal`) and the service template file (`helloWorld_sample_service.bal`) are generated inside the service directory.
+Once you execute the command, the service template file (`helloWorld_sample_service.bal`) are generated inside the service directory.
```

Comment:
```suggestion
Once you execute the command, the service template file (`helloWorld_sample_service.bal`) gets generated inside the service directory.
```

Reply:
Can change as "Once you execute the command, the service template file (`helloWorld_sample_service.bal`) is generated inside the service directory."